bounds.py

```diff
@@ -579,18 +652,20 @@ def normalize(arr):
     return arr / (norm + 1e-12)
 
 
-def compute_ryu_2019(X, n, n_iter=100, eps=1e-8, return_time=True):
-    """Estimate spectral norm of convolutional layer with Ryu2019.
+def compute_ryu_2019(X, n, n_iter=100, return_time=True):
+    """Estimate spectral norm of convolutional layer Ryu2019.
 
     From a convolutional filter, this function estimates the spectral norm of
-    the convolutional layer using Ryu2019 [1]_ or Farnia2019.
+    the convolutional layer for zero padding using Ryu2019 [1]_ or Farnia2019.
 
     Code adapted from [2]_.
 
     Parameters
     ----------
     X : ndarray, shape (cout, cint, h, w)
         Convolutional filter.
+    n : None | int, default=None
```

Comment:
Default value is not None.
```suggestion
    n : None | int
```